frontend/src/app/testResults/resultsTable/ResultsTable.test.tsx

```diff
@@ -123,24 +124,18 @@ describe("Component ResultsTable", () => {
       />
     );
 
-    // 5 rows -> 9 rows
-
     for (const result of TEST_RESULTS_MULTIPLEX_CONTENT) {
-      const resultId = result.internalId;
-
-      for (const multiplexResult of result.results as MultiplexResults) {
-        const testId = `test-result-${resultId}-${toLowerCaseHyphenate(
-          multiplexResult.disease.name
-        )}`;
-        expect(screen.getByTestId(testId)).toBeInTheDocument();
-      }
+      const testId = `test-result-${result.id}-${toLowerCaseHyphenate(
+        result.disease
+      )}`;
+
+      expect(screen.getByTestId(testId)).toBeInTheDocument();
     }
 
-    // TODO: all expected elements are present, but is it exhaustive?
-    // Test to check the # of rows
     expect(screen.getByTestId("filtered-results").children.length).toBe(9);
   });
 
+  /*
```

Comment:
this test has been commented out, please uncomment or remove if it doesnt apply anymore

Reply:
👍 removed in https://github.com/CDCgov/prime-simplereport/pull/6682/commits/34346c5a58c35cca7a4fde47695fcf9e994ce8ae.